server/utils/fieldValidation.js

```diff
@@ -63,18 +63,18 @@ function createSchemaFromConfig(pageConfig) {
     indeterminateCheck: (requiredItem = 'indeterminate', requiredAnswer = 'true') =>
       joi.when(requiredItem, {
         is: requiredAnswer,
-        then: joi.date().format('D/M/YYYY').min(today).max(threeYears).required().messages({
+        then: joi.date().format('DD/MM/YYYY').min(today).max(threeYears).required().messages({
           'date.format': 'The review date must be a real date',
           'date.max': 'The date that they are reviewed by must be within 3 years',
           'date.min': 'The review date must be today or in the future',
         }),
-        otherwise: joi.date().format('D/M/YYYY').min(today).max(oneYear).required().messages({
+        otherwise: joi.date().format('DD/MM/YYYY').min(today).max(oneYear).required().messages({
           'date.format': 'The review date must be a real date',
           'date.max': 'The date that they are reviewed must be within the next 12 months',
           'date.min': 'The review date must be today or in the future',
         }),
       }),
-    todayOrPastDate: joi.date().format('D/M/YYYY').max(today).required().messages({
+    todayOrPastDate: joi.date().format('DD/MM/YYYY').max(today).required().messages({
```

Comment:
the tests now pass again, and I cannot find any front end code that uses D/M/YYYY but still, there is a risk.